### ISSUE TITLE
Review 3

### ISSUE THREAD

**Issue by vr8hub** [hlund05/jules-verne_the-special-correspondent_lovell-coryell-and-company#3]
- [x] Commit 34ad5f3 again has multiple types of work in it, and this time it's mixing editorial (modernizing spelling) and non-editorial (removing italics on non-English words in M-W), and so has to be split. This is why we emphasize one type of work per commit so often—rebasing to correct this is a hassle. Since there are only two commits after this one, and you're not going to need one of them and the other one is just the result of `se clean`, then it might be easier to just drop the last three commits and then redo the modernize spelling and remove italics into two commits. That's up to you; let me know if you need any help. (Don't forget to make a backup of your local repository before starting!)
- [x] Italicizing words/phrases has different rules than italicizing publications. In the former, ending punctuation goes inside the italics if the whole phrase is standalone and doesn't if it isn't (SEMoS 8.2.4/5 discuss this, and lint's t-017 error tries to catch instances of it being done incorrectly). In the case of publications, however, ending punctuation is not part of the actual publication title, and therefore doesn't go inside the italics. So, the period following the book title in ch. 9 should not be inside the italics.
- [x] The "Adding for the Baron’s benefit" line at the end of ch. 9 is not indented in the scans, i.e. is a continuation of the "And in a voice…" paragraph, so should have the "continued" class on it.
- [x] Per the scans, the ch. 13 blockquote should be centered.
- [x] As mentioned earlier, we don't italicize/tag proper names, so Monsieur de Monde in ch. 18 does not need italics.
- [x] Dolce far nienta is in M-W, so it also doesn't need italics.
- [x] As I mentioned at the end of my last comment on Review 2, the CSS for `blockquote p` is not what we want there. We do want a margin between the header and the text, but that should go in the header CSS (and should be 1em). Also, we don't want to _generically_ eliminate text indenting, only on those blockquotes that are centered.
- [x] For the blockquote header CSS, the margin-right isn't necessary; aligned right is fine.

**Comment by hlund05** [comment]
Thanks, I've made it through the feedback above, with just one note: In the story, the German baron is racing around the globe, and in Chap 18 the narrator jokingly refers to him as "Monsieur le Baron Tour de Monde."  I had this in italics since it wasn't his proper name per se, but I've removed the italics in a separate commit -- if this context makes any difference, let me know and I can drop that commit.

**Comment by vr8hub** [comment]
Sorry it's taken so long, I have family in town and my spare time has been almost nil this week.
Right, a proper name is a proper name, even if it's a fake one, so no, that still shouldn't have tags.
Minor last things:
- [x] We only modernize spelling on soundalike words, and kandijar and khanjar are not soundalike, so that change will need to be reverted.
- [x] Semanticate adds an eoc class in ch. 2 that should be accepted. (Its other change should not be.)
- [ ] Have we discussed globe-trotter? Modernize spelling wants to change that to globetrotter; is there a reason it hasn't been accepted?

**Comment by hlund05** [comment]
No worries at all, hope you had a good week.  Comments on your last 3 points:

1. That makes sense, I've reverted the spelling on 'kandijar'
2. OK thanks - added the eoc class in Chap 2
3. I've been back and forth on the term 'globe-trotter' (and 'globe-trotting').  In the PG text it appears variously as one word, two words, and hyphenated.  But in the scans it's consistently hyphenated and in M-W it's hyphenated, so I standardized on that throughout.  However, you're right that modernize-spelling wants to make globetrotter one word (but not globe-trotting).  Would you suggest accepting the changes per modernize-spelling?

**Comment by vr8hub** [comment]
Good catch; if M-W says it's globe-trotter, that's what we'll go with, unless Alex wants to override it. If not, we'll need to remove it from our words file. All right, I believe this is ready to go, I'll hand it over to Alex on the list.
